src/commands/utility/render.js

```diff
@@ -0,0 +1,61 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../')
+
+module.exports = class Render extends Command {
+  constructor (client) {
+    super({
+      name: 'render',
+      aliases: ['show'],
+      category: 'utility',
+      requirements: { guildOnly: true, botPermissions: ['EMBED_LINKS'] },
+      parameters: [{
+        type: 'messageLink', sameGuildOnly: true, forceExists: true, missingError: 'commands:move.missingMessageLink'
+      }]
+    }, client)
+  }
+
+  async run ({ author, t, message }, link) {
+    try {
+      if (!link.channel.permissionsFor(this.client.user.id).has('VIEW_CHANNEL')) throw new CommandError(t('commands:render.iDontHavePermissionToRead'))
```

Comment:
Why you're throwing a error in try-catch block and handling it throwing another error with another message?

Reply:
Fixed in commit 8b04738